allauth_2fa/adapter.py

```diff
@@ -11,7 +11,7 @@ def login(self, request, user):
 
         # Require two-factor authentication if it has been configured.
         if user.totpdevice_set.all():
-            request.session['user_id'] = user.id
+            request.session['allauth_2fa_user_id'] = user.id
```

Comment:
I guess it would reset all login flows, so users that are at the 2fa part of the login flow would be redirected to the login page and would have to enter their credentials again.